R/rcpp.R

```diff
@@ -58,22 +63,17 @@ use_rcpp_eigen <- function(name = NULL) {
 #' @rdname use_rcpp
 #' @export
 use_c <- function(name = NULL) {
-  use_src()
+  check_is_package("use_c()")
+  check_uses_roxygen("use_c()")
 
-  if (!is.null(name)) {
-    name <- slug(name, "c")
-    check_file_name(name)
+  use_src()
```

Comment:
Just some mundane maintenance/refactoring. Makes `use_c()` more similar to `use_rcpp()` and means if these checks fail, the error message makes more sense.

Reply:
This makes perfect sense.